### DISCUSSION TITLE
nome de usuario

### DISCUSSION THREAD

**Discussion by tadkn** [community/community#36355] [New to GitHub]
boa noite agora pouco editei algumas informaçoes do meu perfil porem estava querendo mudar o nome de usuario principal no caso (tadkn) porem apesar de conseguir editar algumas coisas eu notei que o principal que eu quero mudar nao mudou e quero saber o porque nao da pra mudar e quero saber se as pessoas que criaram o github pretendem mudar esse sistema

**Comment by radar-blip** [comment]
If I understand correctly, you want to change your username. You can do that in your `settings` under `account`, like below:
It is not recommended because it can have unintened side affects.
You should be ok since you haven't been a member very long (not much could happen).


![Screenshot from 2022-10-16 22-51-19](https://user-images.githubusercontent.com/39196767/196091731-0eb055f5-b5f4-4eaf-b570-337a0343d3ca.png)

**Reply by tadkn** [reply]
you say it can have side effects.. like what for example

**Reply by swissspidy** [reply]
See https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username for things to keep in mind when changing usernames

**Reply by radar-blip** [reply]
Simple things like in the link @swissspidy provided.
Notifications, links, wont automatically redirect to your new username.
For example, notifications will still go to your old username.

**Comment by ossanna16** [comment]
Hey there @tadkn and thank you for asking a good question! I'm glad you found our community 🙂
